src/assets/modifierdata/engineer.yaml

```
@@ -1,338 +1,355 @@
+      defaultEnabled: true
+
+    - id: big-boomer
+      text: Big Boomer
+      subText: 100%
+      modifiers:
+        damage:
+          Strike Damage: [10%, mult]
+      gw2id: 1947
+      defaultEnabled: true
+
+- section: Firearms
+  id: 38
+  items:
+
+    - id: sharpshooter
+      text: Sharpshooter
+      subText: per hit (including non critical)
+      minor: true
+      amountData:
+        label: 'hit/s'
+        default: 0
+        quantityEntered: 1
+      modifiers:
+        conversionAfterBuffs:
+          Bleeding Coefficient: {Critical Chance: 100%}
+      gw2id: 515
+      defaultEnabled: false
+
+    - id: chemical-rounds
+      text: Chemical Rounds
+      modifiers:
+        attributes:
+          Condition Damage: [120, buff]
+      gw2id: 1878
+      defaultEnabled: true
+
+    - id: high-caliber
+      text: High Caliber
+      modifiers:
+        attributes:
+          Critical Chance: 15%
+      gw2id: 1914
+      defaultEnabled: true
+
+    - id: hematic-focus
+      text: Hematic Focus
+      modifiers:
+        attributes:
+          Critical Chance: 10%
+      minor: true
+      gw2id: 536
+      defaultEnabled: true
+
+    - id: thermal-vision
+      text: Thermal Vision
+      modifiers:
+        damage:
+          Condition Damage: [5%, add]
+        attributes:
+          Expertise: [150, converted]
+      wvwModifiers:
+        damage:
+          Condition Damage: [5%, add]
+        attributes:
+          Expertise: [60, converted]
+      gw2id: 2006
+      defaultEnabled: true
+
+    - id: no-scope
+      text: No Scope
+      amountData:
+        label: '% melee'
+        default: 100
+        quantityEntered: 100
+      modifiers:
+        attributes:
+          Ferocity: [150, buff]
+      gw2id: 1923
+      defaultEnabled: true
+
+    - id: serrated-steel
+      text: Serrated Steel
+      modifiers:
+        attributes:
+          Bleeding Duration: 33%
+      wvwModifiers:
+        attributes:
+          Bleeding Duration: 15%
+      minor: true
+      gw2id: 516
+      defaultEnabled: true
+
+    - id: modified-ammunition
+      text: Modified Ammunition
+      amountData:
+        label: 'conditions'
+        default: 10
+        quantityEntered: 1
+      modifiers:
+        damage:
+          Strike Damage: [2%, mult]
+      gw2id: 526
+      defaultEnabled: true
+
+    - id: incendiary-powder
+      text: Incendiary Powder
+      modifiers:
+        attributes:
+          Burning Duration: 33%
+      wvwModifiers:
+        attributes:
+          Burning Duration: 10%
+      gw2id: 433
+      defaultEnabled: true
+
+- section: Inventions
+  id: 47
+  items:
+
+    - id: over-shield
+      text: Over Shield
+      modifiers:
+        damage:
+          Damage Reduction: [6.6%, unknown]
+      gw2id: 394
+      defaultEnabled: true
+
+    - id: energy-amplifier
+      text: Energy Amplifier
+      modifiers:
+        attributes:
+          Healing Power: [250, buff]   #this is 250 instead of 120

```

```suggestion
          Healing Power: [250, buff]
```
Oh, yep, great catch!